SwiftInterviewKit/API.swift

```diff
@@ -8,35 +8,41 @@
 
 import UIKit
 
+enum Crud:String {
```

Comment:
I think we should name this enum Method. Can you place this enum inside the API class?